pkg/cmd/root/help_reference_test.go

```diff
@@ -0,0 +1,27 @@
+package root
```

Comment:
this file was soley for ease of use testing purposes really, though technically we are validating some basic errors are not occurring on command construction and help generation.

I'm fine removing it if you like